modules/accessing-microshift-cluster-remotely.adoc

```diff
@@ -1,48 +1,48 @@
 // Module included in the following assemblies:
 //
-// microshift/microshift-install-rpm.adoc 
+// microshift/microshift-install-rpm.adoc
 
 :_content-type: CONCEPT
 [id="accessing-microshift-cluster-remotely_{context}"]
-= Accessing the MicroShift cluster remotely
+= Accessing the {product-title} cluster remotely
 
-Use the following procedure to access the MicroShift cluster from a remote workstation. 
+Use the following procedure to access the {product-title} cluster from a remote workstation.
 
-.Prerequisites 
+.Prerequisites
 
-* You have installed the `oc` CLI. 
+* You have installed the the `oc` binary.
 
-.Procedure 
+.Procedure
 
 [NOTE]
 ====
-The following procedure assumes that there are two users. The first user, `user@microshift`, is the user on the {product-title} host machine. `user@microshift` is responsible for setting up the `op-system` machine so that it can be accessed by a remote user on a separate workstation.  
+The following procedure assumes that there are two users. The first user, `user@microshift`, is the user on the {product-title} host machine. `user@microshift` is responsible for setting up the `op-system` machine so that it can be accessed by a remote user on a separate workstation.
 
-The second user, `user@workstation`, accesses the {op-system} machine remotely from their workstation. 
+The second user, `user@workstation`, accesses the {op-system} machine remotely from their workstation.
 ====
 
-. As `user@microshift` on the MicroShift host, open the firewall port for Kubernetes API server (`6443/tcp`) on your {op-system} machine by entering the following command:
+. As `user@microshift` on the {product-title} host, open the firewall port for Kubernetes API server (`6443/tcp`) on your {op-system} machine by entering the following command:
```

Comment:
```suggestion
. As `user@microshift` on the {product-title} host, open the firewall port for Kubernetes API server (`6443/tcp`) by entering the following command:
```

nit: We could remove the redundant phrase indicating which host we expect the user to be on.